pkgs/development/tools/build-managers/scons3/setup-hook.sh

```diff
@@ -0,0 +1,84 @@
+sconsBuildPhase() {
```

Comment:
is this a copy of the existing setup hook?

Reply:
yes.